py/src/app/pcasts/dao/followings_dao.py

```diff
@@ -0,0 +1,44 @@
+from app.pcasts.dao import users_dao
+from . import *
+
+def get_followings(user_id):
+  followings = \
+    Following.query.filter(Following.follower_id == user_id).all()
+
+  return followings
+
+def get_followers(user_id):
+  followers = \
+    Following.query.filter(Following.followed_id == user_id).all()
+
+  return followers
+
+def create_following(follower_id, followed_id):
+  if int(follower_id) != int(followed_id):
+    following = Following(follower_id=follower_id, followed_id=followed_id)
+    follower = users_dao.get_user_by_id(follower_id)
```

Comment:
Rather than 2 DB hits, maybe write a helper to grab users given an array of IDs + then check to see if the length is 2 below - we'll need this helper function in the future anyway